index.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
     'mocha/no-pending-tests': 'error',
     'mocha/no-skipped-tests': 'warn',
     'mocha/valid-test-description': 'error',
+    'no-template-curly-in-string': 'off',
```

Comment:
I'm fine with ignoring that rule on the Bunsen schemas.  Remove it and you won't get lint errors on trying to use templates with the back ticks which is useful.